packages/dx-react-chart/src/plugins/axis.tsx

```diff
@@ -20,7 +21,7 @@ const SVG_STYLE = {
   position: 'absolute', left: 0, top: 0, overflow: 'visible',
 };
 
-const adjustScaleRange = (scale, [width, height]) => {
+const adjustScaleRange = (scale: Scale, [width, height]: [number, number]) => {
```

Comment:
There is `Size` type in *dx-react-core*.

Reply:
The `Size` in `dx-react-core` is object, not the array

Reply:
But I have find several places were such array using, thus I'll create extra type for that.